lib/was/thumbnail_service/picker/memento_picker_threshold_grouping.rb

```diff
@@ -2,7 +2,7 @@ module Was
   module ThumbnailService
     module Picker
       class MementoPickerThresholdGrouping
-         @threshold = 0
+        @threshold = Rails.configuration.respond_to?(:threshold) ? Rails.configuration.threshold : 0
```

Comment:
@cbeer Any idea how can I test this line? The value is loaded in the boot time and I can't change it in the spec like this
```
  it 'reads the default threshold value as 0' do
    Rails.configuration.threshold = nil
    puts Was::ThumbnailService::Picker::MementoPickerThresholdGrouping.instance_variable_get(:@threshold)
  end
  it "reads the threshold configuration" do
    Rails.configuration.threshold = 5
    puts Was::ThumbnailService::Picker::MementoPickerThresholdGrouping.instance_variable_get(:@threshold)
  end
```